pql/parser.go

```diff
@@ -28,6 +28,7 @@ const timeFormat = "2006-01-02T15:04"
 
 // duplicateArgErrorMessage is used as an error string in the parser.
 const duplicateArgErrorMessage = "duplicate argument provided"
+const parsingIntErrorMessage = "out of bounds"
```

Comment:
This is minor, and I'm not sure if we have any solid conventions about this, but I'd suggest matching the name and message of this error, maybe `outOfBoundsErrorMessage`. See https://github.com/pilosa/pilosa/blob/master/pilosa.go#L26 for the de facto convention.